.env.example

```
@@ -10,8 +10,8 @@ GITHUB_ORG=
 
 UNCOMMITTED_CHANGES_POLL_INTERVAL_MS=5000
 
-FRONTEND_BASE_URL=http://localhost:3010
-
+FRONTEND_BASE_URL=http://localhost:3005
```

```suggestion
FRONTEND_BASE_URL=http://localhost:3010
```
(republik-)frontend is correct at 3010 by default